compose/cli/main.py

```diff
@@ -264,18 +264,29 @@ def create(self, options):
 
     def down(self, options):
         """
-        Stop containers and remove containers, networks, volumes, and images
-        created by `up`. Only containers and networks are removed by default.
+        Stops containers and removes containers, networks, volumes, and images
+        created by `up`.
+
+        By default, the only things removed are:
+
+        - Containers for services defined in the Compose file
+        - Anonymous volumes attached to those containers
+        - Networks defined in the `networks` section of the Compose file
+        - The default network, if one is used
+
+        Networks and volumes defined as `external` are never removed.
 
         Usage: down [options]
 
         Options:
-            --rmi type          Remove images, type may be one of: 'all' to remove
-                                all images, or 'local' to remove only images that
-                                don't have an custom name set by the `image` field
-            -v, --volumes       Remove data volumes
-            --remove-orphans    Remove containers for services not defined in
-                                the Compose file
+            --rmi type          Remove images. Type must be one of:
+                                'all': Remove all images used by any service.
+                                'local': Remove only images that don't have a custom tag
+                                set by the 'image' field.
+            -v, --volumes       Remove named volumes declared in the `volumes` section
+                                of the Compose file.
```

Comment:
Add info as per [comment](https://github.com/docker/compose/issues/2838#issuecomment-181569292):
Note: compose cannot remove volumes after a call of down w/o specifing -v flag.